.circleci/config.yml

```diff
@@ -690,6 +690,7 @@ jobs:
           name: Check frozen files
           command: just check-frozen-code
           working_directory: packages/contracts-bedrock
+          if: ${{ !contains(github.event.pull_request.labels.*.name, 'S-exempt-frozen-files') }}
```

Comment:
This looks like [github actions syntax](https://stackoverflow.com/questions/59588605/how-to-check-for-a-label-in-a-github-action-condition), not circleci?

I think you're probably going to need to do something like what @mds1 did here: 51dc3bdcf5899fe2a876322b79d611e059ad578c

Reply:
Right, rewritten it following the example just above.